### PR TITLE
Add pg_postmaster_start_time

### PR DESCRIPTION
**Pull request by mfussenegger**
## Summary of the changes / Why this improves CrateDB

To make datagrip a happier.

## Checklist

 - [x] Added an entry in `CHANGES.txt` for user facing changes
 - [x] Updated documentation & `sql_features` table for user facing changes
 - [x] Touched code is covered by tests
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

**Review by seut — APPROVED** [review]
👍

File: docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -55,6 +55,9 @@ Deprecations
 Changes
 =======
 
+- Added the :ref:`pg_postmaster_start_time() <scalar_pg_postmaster_start_time>`
+  scalar function.
```
Comment (norosa) [inline docs/appendices/release-notes/unreleased.rst:59]:
```suggestion
  :ref:`scalar function <scalar-functions>`.
```

**Review by norosa — COMMENTED** [review]
lgtm

File: docs/appendices/release-notes/unreleased.rst (hunk 1 above)
Comment (mfussenegger) [reply docs/appendices/release-notes/unreleased.rst:59]: Sorry, mergify already merged before I saw this. Do you have a strong opinion on this? I tend to prefer only one link in this case as it's the same document anway and the users don't have insights into where the links are pointing to - so they might be inclined to click on both and discover that it's the same thing anyways

**Pull request merged**